wdae/wdae/chromosome/views.py

```diff
@@ -17,17 +17,17 @@ def __init__(self):
         reader = pd.read_csv(csvfile, delimiter='\t')
```

Comment:
this is already parsed data frame so it probably better to rename it something 'chrom_df' or similar